scripts/validate/gomod.sh

```diff
@@ -12,8 +12,6 @@ else
   echo "go mod tidy passed."
 fi
 
-git diff --exit-code go.mod go.sum || exit_code=$?
```

Comment:
This has literally only provided headaches over the time. I don't see any good reason to have it.